data/sources.txt

```diff
@@ -1,14 +1,14 @@
-https://gist.githubusercontent.com/Artistan/9662757/raw/896006657dfea1207a1ecbe7ee8b339b26550ee1/free_email_provider_domains.txt
-https://gist.githubusercontent.com/agarstang/0d87cae417f25a0b90f3/raw/07ed191d57a161699ba921178bd4712429ec91e0/free_email_provider_domains.txt
-https://gist.githubusercontent.com/cnsaturn/9919758/raw/1229d5d72a8c1b3c893129c67b3826b8f1e34040/3rd_party_email_provider_domains.txt
-https://gist.githubusercontent.com/com4/ff0711c7f49bd792950a/raw/1302abaf2c01854980cf8d64c672c568c92cbc81/free_email_provider_domains.txt
-https://gist.githubusercontent.com/cyriac/f89634a28f4d441719d8/raw/43545188d525cd1581ecbd50adf0a1ac32a1feb2/free_email_provider_domains.txt
-https://gist.githubusercontent.com/defeated/6500068/raw/01733a71051bce3db6eccc736f256d40d7ec63b8/free_email_provider_domains.txt'
-https://gist.githubusercontent.com/hadees/3cc0e2cf97d06e0b8ebb/raw/40c7dcad6e8b819c89f89dcd78bb62ab709d5e53/free_email_provider_domains.txt
-https://gist.githubusercontent.com/jpadilla/8459489/raw/e6fcc062d62c005163820bb02a5c7315ad7a43a6/free_email_provider_domains.txt
-https://gist.githubusercontent.com/nikhilben/d4797617901c20dbc5de/raw/96183d95a805ee9b138bb043c92f859e6428d091/List%20of%20free%20email%20domains
-https://gist.githubusercontent.com/rasmussvanejensen/3a361d113864ef35eafb/raw/e6fcc062d62c005163820bb02a5c7315ad7a43a6/free_email_provider_domains.txt
-https://gist.githubusercontent.com/tbrianjones/5992856/raw/bede1836eac8102bb7d7f966ef7e690d58ce5252/free_email_provider_domains.txt
+https://gist.githubusercontent.com/Artistan/9662757/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/agarstang/0d87cae417f25a0b90f3/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/cnsaturn/9919758/raw/3rd_party_email_provider_domains.txt
+https://gist.githubusercontent.com/com4/ff0711c7f49bd792950a/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/cyriac/f89634a28f4d441719d8/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/defeated/6500068/raw/free_email_provider_domains.txt'
+https://gist.githubusercontent.com/hadees/3cc0e2cf97d06e0b8ebb/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/jpadilla/8459489/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/nikhilben/d4797617901c20dbc5de/raw/List%20of%20free%20email%20domains
+https://gist.githubusercontent.com/rasmussvanejensen/3a361d113864ef35eafb/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/tbrianjones/5992856/raw/free_email_provider_domains.txt
+https://gist.githubusercontent.com/adamloving/4677212/raw/common-email-providers.txt
 https://raw.githubusercontent.com/aaronbassett/DisposableEmailChecker/master/disposable_email_domains.txt
 https://raw.githubusercontent.com/tarr11/Webmail-Domains/master/domains.txt
-https://gist.githubusercontent.com/adamloving/4677212/raw/8f708b145be0e546239dc6e7bb4f4dc26ef49856/common-email-providers.txt
```

Comment:
Yeah exactly that, at least as far as I understand it. You were referencing a specific hash instead of the lastest version.

And as a bonus it's shorter this way.